classes/migration/upgrade/v3_4_0/I7725_DecisionConstantsUpdate.php

```diff
@@ -0,0 +1,207 @@
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 3,
+            'updated_value' => 5,
+        ],
+
+        // \PKP\decision\Decision::DECLINE
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 4,
+            'updated_value' => 6,
+        ],
+
+        // \PKP\decision\Decision::INITIAL_DECLINE
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_SUBMISSION],
+            'current_value' => 9,
+            'updated_value' => 8,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_ACCEPT
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 11,
+            'updated_value' => 9,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_PENDING_REVISIONS
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 12,
+            'updated_value' => 10,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_RESUBMIT
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 13,
+            'updated_value' => 11,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_DECLINE
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 14,
+            'updated_value' => 12,
+        ],
+
+        // \PKP\decision\Decision::NEW_EXTERNAL_ROUND
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 16,
+            'updated_value' => 14,
+        ],
+
+        // \PKP\decision\Decision::REVERT_DECLINE
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 17,
+            'updated_value' => 15,
+        ],
+
+        // \PKP\decision\Decision::REVERT_INITIAL_DECLINE
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_SUBMISSION],
+            'current_value' => 18,
+            'updated_value' => 16
+        ],
+
+        // \PKP\decision\Decision::SKIP_EXTERNAL_REVIEW
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EDITING],
+            'current_value' => 19,
+            'updated_value' => 17,
+        ],
+
+        // \PKP\decision\Decision::BACK_FROM_PRODUCTION
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_EDITING],
+            'current_value' => 31,
+            'updated_value' => 29,
+        ],
+
+        // \PKP\decision\Decision::BACK_FROM_COPYEDITING
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_SUBMISSION, WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 32,
+            'updated_value' => 30,
+        ],
+
+        // \PKP\decision\Decision::CANCEL_REVIEW_ROUND
+        [
+            'stage_id' => [WORKFLOW_STAGE_ID_SUBMISSION, WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],
+            'current_value' => 33,
+            'updated_value' => 31,
+        ],
```

Comment:
Up to you if you want to leave them in, but these shouldn't exist in any version before 3.4.

Reply:
yes it should not exist but I put those just to make sure that if any of us devs wont get in trouble for the update with current submission as our own installation . For example , I already have several submission in my own installation with those decisions and once this get merged , if those updates not pushed , it will have unsynced decision value . But if you think it's better to remove, I will do it.

Reply:
Yeah makes sense. I'm ok to leave it in.